hangups/conversation.py

```diff
@@ -679,8 +679,7 @@ def next_event(self, event_id, prev=False):
             return self.events[i - 1]
         elif not prev and i + 1 < len(self.events):
             return self.events[i + 1]
-        else:
-            return None
+        return None
```

Comment:
I think I prefer the existing `if else` style over what Pylint wants to enforce. PEP8 says that either is fine.